src/app/sidenav/sidenav.component.ts

```diff
@@ -16,7 +16,7 @@ export class SidenavComponent implements OnInit {
 
   // I'm not sure if 1000px is a great cutoff point - seems a bit too big (maybe decrease to ~768px?)
   get isMobile(): boolean {
```

Comment:
Please move the `isMobile` method to an external class and make it a static method.